data_processors/pipeline/lambdas/wes_handler.py

```diff
@@ -175,3 +183,21 @@ def _extended_get_workflow_run(wfr_id, wfr_event, run_api):
     result = {'status': _status, 'end': _end, 'output': _output}
     logger.info(libjson.dumps(result))
     return result
+
+
+def get_gds_work_dir(subpath):
```

Comment:
I'd recommend, we also move this to `data_processors/pipeline/domain/workflow.py` and possible under `WorkflowHelper`. Or ditto more specific `EngineParametersHelper(Helper)` model if you like.

Reply:
Functions now redundant with introduction of EngineParametersHelper